doc/manual/introduction.md

```diff
@@ -74,9 +74,9 @@ website.
 
 ## Current state and roadmap
 
-Nickel is currently released in version `1.4`. The core design of the language
```

Comment:
To avoid getting out of sync every version, I've just reworded this part to not mention the current version. I think the main message to convey is that Nickel is post 1.0 and that you should use it.